src/shared/api/urls.ts

```diff
@@ -173,10 +173,6 @@ export function getUniprotIdUrl(swissProtAccession: string) {
     });
 }
 
-export function getMutationAlignerUrl() {
```

Comment:
Seems like this function was not used anymore.  Currently it always points to the public instance. Do we still need to worry about making the proxy url customizable per instance? Since this feature is not very important, maybe it is okay to just remove this.

Reply:
Actually, we decided to keep this to be able to use instance specific proxy URLs.